### PR TITLE
added js1 week 4

### PR DESCRIPTION
**Pull request by STEP0HEN**

### CODE REVIEW

File: javascript/javascript1/week4/week4-homework/getcommand.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+function getReply(command) {
+    
+    //Saving the person's name and returning nice to meet you plus the person's name
+    if (command.startsWith('Hello') && command.includes('name')) {
+          let commandArray = command.split(' ')
+          let name = commandArray[commandArray.length -1]
+          names.push(name)
+        
+        return `Nice to meet you ${names[names.length -1]}`;
```
Comment (flom84) [inline javascript/javascript1/week4/week4-homework/getcommand.js]:
```suggestion
        return `Nice to meet you ${name}`;
```
No need to access the array again :) ...One of the most important things you need to remember in software development is not to repeat yourself (google *DRY principle* if you want to learn more about it :) )

File: javascript/javascript1/week4/week4-homework/getcommand.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+let whoAmI = 'What is my name';
+let day = 'What day is it today'
+let todo =[];
+let myTodoList = 'What is on my todo'
+let favDish = [];
+let event = [];
+let eventList = 'What am I doing this week'
+
+
+function getReply(command) {
```
Comment (flom84) [inline javascript/javascript1/week4/week4-homework/getcommand.js:15]: What would happen if I call this function two times like this ?
`console.log(getReply('Hello my name is Benjamin'))
console.log(getReply('Hello my name is Benjamin'))
`
Can you spot the problem ?

File: javascript/javascript1/week4/week4-homework/getcommand.js (hunk 1 above)
Comment (STEP0HEN) [reply javascript/javascript1/week4/week4-homework/getcommand.js]: okay.

File: javascript/javascript1/week4/week4-homework/getcommand.js (hunk 2 above)
Comment (STEP0HEN) [reply javascript/javascript1/week4/week4-homework/getcommand.js:15]: It'll log out 'nice to meet you Benjamin' 2x
I see the problem, I'll fix it.

**Pull request merged**